content/sensu-go/6.5/sensuctl/environment-variables.md

```diff
@@ -12,20 +11,72 @@ menu:
     parent: sensuctl
 ---
 
-Sensuctl includes the `sensuctl env` command to help export and set environment variables on your systems.
+Sensu allows you to set sensuctl environment variables for a [single sensuctl command][1], with [sensuctl configure][2], or with [sensuctl env][3].
```

Comment:
I think we should acknowledge that you can do all this configuration with flags instead, but I'm not sure why a customer would use one or the other. Is it just a matter of preference? Or are there functionality differences between flags and env vars that we need to point out here?

Reply:
Functionality is the same as using the flags. As for the why question I can see these scenarios:
* customer has two different environments - having one shell with a set of variables and another one with another set prevents you from having to run sensuctl configure every time you go from one to the other
* security-wise if someone intercepts your `sencuctl configure` command, if you're using flags the sensitive information is visible whereas it's not if using environment
* environment is also easier to pass into a container creation script

I'm sure @calebhailey is able to provide additional information on the operational side of Sensu that makes this useful.